app/helpers/user_helper.php

```diff
@@ -10,4 +10,12 @@ function objectToArray($obj){
         $array[$key] = $value;
     }
     return $array;
-}
\ No newline at end of file
+}
+
+function securedPage()
+{
+    $is_logged = isset($_SESSION['username']);
+    if (!$is_logged) {
+        redirect('/');
```

Comment:
this is just a helper functions. but yet you have some redirect? :scream::scream:
hmm.. i think redirect functions should be done in a controller class only